examples/convert_mri_and_trans.py

```diff
@@ -134,16 +135,20 @@
 anat_dir = t1w_bids_path.directory
 
 ###############################################################################
-# Let's have another look at our BIDS directory
+# Let's have another look at our BIDS directory and plot the written landmarks
 print_dir_tree(output_path)
 
+plot_anat_landmarks(t1w_bids_path, vmax=160)
+plt.suptitle('T1 MRI')
+
 ###############################################################################
 # Our BIDS dataset is now ready to be shared. We can easily estimate the
-# transformation matrix using ``MNE-BIDS`` and the BIDS dataset.
+# transformation matrix using ``MNE-BIDS`` and the BIDS dataset since we
+# have now anatomical landmarks.
 estim_trans = get_head_mri_trans(bids_path=bids_path)
 
 ###############################################################################
-# Finally, let's use the T1 weighted MRI image and plot the anatomical
+# One can also directly use the T1 weighted MRI image and plot the anatomical
 # landmarks Nasion, LPA, and RPA onto the brain image. For that, we can
 # extract the location of Nasion, LPA, and RPA from the MEG file, apply our
 # transformation matrix :code:`trans`, and plot the results.
```

Comment:
Wondering if we should start off this paragraph differently or put it into a new subsection. The first sentence made me think, "umm, but we just DID that??" until I got to the point where it says, " ... from the MEG file"